assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -205,29 +202,26 @@ function Title({
           tabIndex={ariaHidden ? -1 : 0}
         />
       )}
-      <HeaderButton onClick={onToggle}>
-        <Heading id={panelTitleId}>{children}</Heading>
-        <HeaderActions>
-          {secondaryAction}
-          {canCollapse && (
-            <Toggle
-              isCollapsed={isCollapsed}
-              toggle={onToggle}
-              aria-label={titleLabel}
-              aria-expanded={!isCollapsed}
-              aria-controls={panelContentId}
-              tabIndex={ariaHidden ? -1 : 0}
-            >
-              <Arrow />
-            </Toggle>
-          )}
-        </HeaderActions>
-      </HeaderButton>
+      <Heading id={panelTitleId}>{children}</Heading>
+      <HeaderActions>
+        {secondaryAction}
+        {canCollapse && (
+          <Toggle
+            isCollapsed={isCollapsed}
+            aria-label={titleLabel}
```

Comment:
I think you ought to remove this from here?

Reply:
Removed the label, but need to keep isCollapsed for the component style reason.